scripts/delphix-bootcount.service

```diff
@@ -11,7 +11,7 @@ After=ssh.service
 Type=oneshot
 RemainAfterExit=yes
 ExecStartPre=/bin/sleep 300
-ExecStart=/usr/bin/grub-editenv /boot/grub/grubenv set bootcount=0
```

Comment:
While we are there, I think we'd want to avoid having this service run in an upgrade container as well, by adding
```
[Unit]
...
ConditionVirtualization=!container
```